packages/web/src/components/ui/Hero/Hero.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import styles from './Hero.module.scss';
+import Container from '../Container/Container';
+import Chip from '../Chip/Chip';
+import { Arrow } from '../../icons';
+import HeaderNew from '../HeaderNew/HeaderNew';
+
+export type HeroProps = {
+  inputText?: string;
+  loading?: boolean;
+  suggestions: string[];
+};
+
+export default function Hero({ inputText, suggestions, loading = false }: HeroProps) {
```

Comment:
Is this some common name? Sounds misleading to me, never seen this before.

Reply:
Quite common, actually.

Few examples:
github
![image](https://user-images.githubusercontent.com/2999208/186891112-d5e8cbb4-2de5-468e-9dfa-0a1da77fbe34.png)

audible
![image](https://user-images.githubusercontent.com/2999208/186891953-9e67f50f-3b8e-4822-b693-5a8e11bcacc4.png)

vercel
![image](https://user-images.githubusercontent.com/2999208/186892622-9dafc8e5-827a-47c5-a6a3-a8f65de302c6.png)

Reply:
Ok, thanks :) Another day, another interesting fact